docs/workflow.md

```diff
@@ -30,6 +30,24 @@ TODO: Detail routes used to make the workflow advance
 
 ## Details
 
+### Initialization
+
+When initializing a repository with the workflow configured as :
+
+```yaml
+workflow:
+  - master
+  - staging
+  - production
+```
+
+The 3 branches (master, staging, production) will be created.
+You will be able to merge in the first step/branch defined (master here).
+The other branches will be protected so that you cannot merge in (except with higher permissions).
+This requires to have a dedicated account with sufficient permissions (see [Prerequisites](./getting_started.md#github-account-prerequisite)).
```

Comment:
```suggestion
The other branches will be protected so that a merge commit can only be triggered by a workflow.   _This requires to have a dedicated account with sufficient permissions (see [Prerequisites](./getting_started.md#github-account-prerequisite))._
```